src/diy/smol-slime.md

```diff
@@ -1,640 +1,24 @@
-| -------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------ |
-| SuperMini nRF52840         | The most affordable option overall. It is a clone of the nice!nano board. It is available on AliExpress with `compatible with nice!nano` or `Pro Micro` branding. |
-| Seeed Studio XIAO nRF52840 | Compact board available at Seeed Studio. |
-
-### Inertial Measurement Units
- - BMI270
- - ICM-42688-P
- - ICM-42688-V
- - ICM-45686
- - ISM330BX
- - ISM330DHCX
- - ISM330DLC
- - LSM6DS3
- - LSM6DS3TR-C
- - LSM6DSL
- - LSM6DSM
- - LSM6DSO
- - LSM6DSR
- - LSM6DSV
- - LSM6DSV16B
-
-### Magnetometers
- - AK09940
- - BMM150*
- - BMM350*
- - IIS2MDC
- - IST8306
- - IST8308
- - LIS2MDL
- - LIS3MDL*
- - MMC5983MA
-
-*Sensor driver has not been tested.
-
-### Sensor Modules
+Smol Slimes, also known as nRF Trackers, use a protocol called Enhanced ShockBurst (ESB) on Nordic Semiconductor nRF52 and nRF54L series System-on-Chip (SoC). These are very power-efficient trackers, requiring a much smaller battery that can last for days to weeks compared to traditional ESP (WiFi) SlimeVR Trackers. A receiver, or dongle, is required to bridge communication between trackers and the SlimeVR Server. This method does not work with Quest Standalone without using a computer for OSC.
 
-#### IMU Modules
-Some of the supported sensor modules are described on the [IMU Comparison page](imu-comparison.md). Please note that the most common sensor modules are not supported.
+## Smol Pages
 
-#### IMU + Magnetometer Modules
-Meia, a member of the [SlimeVR Discord](#discord), produces and sells sensor modules with an onboard magnetometer. These modules are compatible with standard sensor modules, and their form factor is designed for stacked builds.
+- [Smol Hardware](smol-slimes/smol-hardware.md)
+- [Smol Firmware](smol-slimes/smol-firmware.md)
+- [Smol Community Cases](smol-slimes/smol-slimes-community-cases.md)
 
-| IMU + Magnetometer                                 | Product Page                                                                 |
-| -------------------------------------------------- | ---------------------------------------------------------------------------- |
-| [ICM-45686](imu-comparison.md#ICM-45686) + IST8306 | [store.kouno.xyz](https://store.kouno.xyz/products/icm-45686-ist8306-module) |
-| LSM6DSR + IST8306                                  | [store.kouno.xyz](https://store.kouno.xyz/products/lsm6dsr-ist8306-module)   |
+```admonish note
+Welcome traveler. If you new and feeling lost - I would suggest to go through [Smol Community Cases](smol-slimes/smol-slimes-community-cases.md) to get examples of finished projects. Community cases pages frequently include list of parts you can use to make your life easier.
```

Comment:
i dont think "community cases pages" is appropriate, nor is it always guaranteed to be easy